docker-black/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.6-alpine3.7
+
+WORKDIR /app
+RUN pip install black
+
+LABEL name=docker-black version=dev \
+      maintainer="Alex Viscreanu <aexvir@kiwi.com>"
+
+ENTRYPOINT [ "black" ]
```

Comment:
Any specific reason for using `ENTRYPOINT`? This is usually more hassle than anything else, I'd do `CMD` here.

Reply:
In order to allow both refactoring and only checking differences black should be launched with different arguments. That's why.